src/misc/location/index.ts

```diff
@@ -0,0 +1,13 @@
+import { HttpClient } from '../../shared/http/client';
+
+export interface UserLocation {
+  ip: string;
+  location: string;
+}
+
+export const getUserLocation = async (apiUrl: string): Promise<UserLocation> => {
+  const client = HttpClient.create(apiUrl);
+  return client.get<UserLocation>(`${apiUrl}`, {
+    method: 'GET',
```

Comment:
it is necessary to specify `method: 'GET'`?

Reply:
Ops, nop, it is not needed.